src/main/java/kr/codesqaud/cafe/controller/UserController.java

```diff
@@ -0,0 +1,58 @@
+package kr.codesqaud.cafe.controller;
+
+import kr.codesqaud.cafe.dto.UserDTO;
+import kr.codesqaud.cafe.repository.MemoryUserRepository;
+import kr.codesqaud.cafe.repository.UserRepository;
+import kr.codesqaud.cafe.service.UserService;
+import kr.codesqaud.cafe.user.User;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Controller
+@RequestMapping("/users")
+public class UserController {
+    private final MemoryUserRepository memoryUserRepository;
+
+    @Autowired
+    public UserController(MemoryUserRepository memoryUserRepository) {
+        this.memoryUserRepository = memoryUserRepository;
+    }
+
+    @GetMapping("/user/form")
+    public String join() {
+        return "user/form";
+    }
+
+    @GetMapping
+    public String listAllUsers(Model model){
```

Comment:
```suggestion
    public String listAllUsers(Model model) {
```
요런 코드 컨벤션을 준수할 수 있도록 IntelliJ의 Reformat 기능을 사용하거나, lint 툴을 사용해보세요.